docs/nl/index.html

```diff
@@ -1,16 +1,16 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml">
    <head><meta charset="UTF-8" />
-      <title>OmegaT 4.2 - Gebruikershandleiding</title>
+      <title>OmegaT 6.0.0 - User Manual</title>
       <link href="OmegaT.css" rel="stylesheet" type="text/css" />
       <meta content="DocBook XSL Stylesheets V1.79.1" name="generator" />
-      <meta content="Dit document is de officiële gebruikershandleiding voor OmegaT, het gratis Computer Aided Translation-programma. Het bevat ook instructies voor de installatie." name="description" />
-      <link href="chapter.installing.and.running.html" rel="next" title="Installeren en uitvoeren van OmegaT" />
+      <meta content="Dit document is de officiële gebruikershandleiding voor OmegaT, het gratis Computer Aided Translation-programma. Het belangrijkste doel van OmegaT is om u te helpen een goede vertaling te maken op de snelst mogelijke manier door u toegang te verschaffen tot de bronnen van uw vertaling op de meest betrouwbare manier. OmegaT wordt gedistribueerd onder de termen van de GNU General Public License, versie 3.0. De licentie geeft u de volgende vrijheden:De vrijheid om OmegaT uit te voeren zoals u dat wilt, voor elk doel.De vrijheid om te bestuderen hoe OmegaT werkt, en het te wijzigen zodat het uw computerwerk uitvoert zoals u dat wilt.De vrijheid om kopieën opnieuw te distribueren, zodat u anderen kunt helpen.De vrijheid om kopieën van uw aangepaste versies te distribueren aan anderen." name="description" />
+      <link href="chapter.instant.start.guide.html" rel="next" title="Introductie voor OmegaT" />
```

Comment:
The file "chapter.instant.start.guide.html" is not in the PR. Need to check missing files.